front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/[dsId]/documents/[documentId]/index.ts

```diff
@@ -472,6 +475,25 @@ async function handler(
         });
       }
 
+      const statsDTags = [
+        `data_source_id:${dataSource.id}`,
+        `workspace_id:${owner.sId}`,
+        `data_source_name:${dataSource.name}`,
+      ];
+      if (!r.data.parents || r.data.parents.length === 0) {
+        statsDClient.increment(
+          "document_without_a_parent.count",
+          1,
+          statsDTags
+        );
+      } else if (r.data.parents[0] != req.query.documentId) {
+        statsDClient.increment(
+          "document_without_a_parent.count",
```

Comment:
Wrong name ?

Reply:
yes, copy pasted from above 🤦 
fixed in https://github.com/dust-tt/dust/pull/9013

Reply:
👍